packages/venia-ui/lib/queries/getProductDetailBySku.graphql

```diff
@@ -38,9 +38,6 @@ query productDetailBySku($sku: String, $onServer: Boolean!) {
                     }
                 }
             }
-            meta_title @include(if: $onServer)
-            meta_keyword @include(if: $onServer)
-            meta_description @include(if: $onServer)
```

Comment:
These were unused. This query is only used for the cart and shouldn't need these details anyways.